packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -121,8 +120,7 @@ test.describe('severity-1 #smoke', () => {
       // We used to have this, not sure if we want it or not.
       // wait for navigation, and get search params
       // await page.waitForURL(/oauth\//);
-      // const url = page.url();
-      // const params = new URLSearchParams(url.substring(url.indexOf('?') + 1));
+      // const params = new URL(page.url()).searchParams;
```

Comment:
Maybe we should just delete these comments?  If the author of the comment is "not sure" then I don't see a reason to keep them.

Reply:
WFM, I'll remove.